servers.yaml

```diff
@@ -4,7 +4,7 @@ default_config:
     run_os_update: true
 servers:
   - name: amp
-    box: bento/centos-7.3
+    box: bento/centos-8.1
```

Comment:
Why the changes to centos 8.1?

Reply:
Because that is what this branch does. Upgrades the boxes we use to 8.1.